### Footstep parameterization for a large humanoid, similar to ASIMO or HRP-2 ##

### footstep settings ##########################################################
# Footstep displacement vectors (in meter / rad)
# WARNING: data type has to be float/double, integers will cause an exception
footsteps:
  x:             [ 0.30,  0.20,  0.10, -0.10,  0.20,  0.10,  0.00,  0.00]
  y:             [ 0.25,  0.25,  0.25,  0.25,  0.30,  0.25,  0.25,  0.25]
  theta:         [ 0.00,  0.00,  0.00,  0.00,  0.25, -0.25,  0.25,  0.05]
  swing_height:  [ 0.15,  0.15,  0.15,  0.15,  0.15,  0.15,  0.15,  0.15]
  step_duration: [ 0.63,  0.63,  0.63,  0.63,  0.63,  0.63,  0.63,  0.63]
  step_cost:     [ 0.05,  0.00,  0.00,  0.00,  0.00,  0.00,  0.00,  0.00]

### step range #################################################################
foot:
  max:
    step: {theta: 0.40}
    inverse:
      step: {theta: -0.30}

# define polygon for valid step position in view of supporting (=standing) foot
step_range:
  x: [ 0.00, -0.20, -0.17, -0.13,  0.13,  0.20,  0.38]
  y: [ 0.20,  0.25,  0.30,  0.35,  0.35,  0.30,  0.20]

# standard params
step_duration: 0.63
sway_duration: 0.1      # Ignored if it is less than 0.2
swing_height:  0.05     # Ignored if it is less than 0.1
lift_height:   0.05     # Ignored if it is less than 0.1 (will be added to swing height in WALK)

use_terrain_model: False

